tests/acosh_complex.cpp

```diff
@@ -26,47 +42,18 @@ template <typename T> struct test_acosh {
       });
     }
     Q.wait();
```

Comment:
Just for elegance for `Q.wait()` should be inside the `if(is_error_checking)`, but not a big deal